var/spack/repos/builtin/packages/abi-dumper/package.py

```diff
@@ -11,10 +11,11 @@ class AbiDumper(MakefilePackage):
     DWARF debug info."""
 
     homepage = "https://github.com/lvc/abi-dumper"
-    url      = "https://github.com/lvc/abi-dumper/archive/1.1.tar.gz"
+    url      = "https://github.com/lvc/abi-dumper/archive/1.2.tar.gz"
 
-    version('1.1',     sha256='ef63201368e0d76a29d2f7aed98c488f6fb71898126762d65baed1e762988083')
-    version('1.0',     sha256='bfa0189a172fa788afc603b1ae675808a57556a77a008e4af8f643d396c34bbb')
+    version('1.2', sha256='8a9858c91b4e9222c89b676d59422053ad560fa005a39443053568049bd4d27e', url='https://github.com/lvc/abi-dumper/archive/1.2.tar.gz')
```

Comment:
I think the `url` can be extrapolated from the one at class level (no need to repeat it in the version directives if the `url` does not change structure).